### DISCUSSION TITLE
how can I customize the background？

### DISCUSSION THREAD

**Discussion by lindaoling** [bcakmakoglu/vue-flow#1495] [Q&A]
how can I customize the background, such as drawing an axis and placing a background image?

**Comment by bcakmakoglu** [comment]
By using your own background? What exactly are you unable to do? Can you be specific?

**Comment by lindaoling** [comment]
![微信截图_20240620143635](https://github.com/bcakmakoglu/vue-flow/assets/9306261/9ba12ef9-9535-411f-9925-e87db221f7da)
I want to draw a coordinate system on the background, and the coordinate system adapts to the background scaling.

I tried to extend '@vue-flow/background' to achieve it, but could not find the corresponding API

if using my own background，i don't know how to find the starting point of the coordinate axis

**Reply by bcakmakoglu** [reply]
If you just want to draw lines you can use `<Background variant="lines" />`

**Reply by lindaoling** [reply]
not lines but a coordinate system

**Reply by bcakmakoglu** [reply]
Not sure what exactly you are trying to achieve, the screenshot shows a background made up of lines (whatever gaps you use idc, you can configure that in the Background cmp), which you can already do.
Idk what you want as a "coordinate" system, your question is vague at best.

**Reply by SidSid1986** [reply]
> variant="lines"

thank a lot，thanks for Solved my problem
